website/src/components/Messages/MessageTree.tsx

```diff
@@ -47,35 +43,34 @@ export const MessageTree = memo(({ tree, messageId }: { tree: MessageWithChildre
   return (
     <>
       <Box position="relative" className="root">
-        <Box
-          className="root-connection"
-          top={{ base: toPx(0), md: 0 }}
-          height={{ base: `calc(100% - ${toPx(8 + avatarSize / 2)})`, md: `100%` }}
-          position="absolute"
-          width="2px"
-          bg={connectionColor}
-          left={toPx(avatarSize / 2 - 1)}
-        ></Box>
-        <Box
-          display={{ base: "block", md: "none" }}
-          position="absolute"
-          height={`calc(100% - ${toPx(6 + avatarSize / 2)})`}
-          width={`10px`}
-          top={toPx(6 + avatarSize / 2)}
-          borderTopWidth="2px"
-          borderTopLeftRadius="10px"
-          left="-8px"
-          borderTopStyle="solid"
-          borderLeftWidth="2px"
-          borderColor={connectionColor}
-          className="root-curve"
-        ></Box>
-        <MessageTableEntry
-          showAuthorBadge
-          message={tree}
-          avartarPosition="top"
-          highlight={tree.id === messageId}
-        ></MessageTableEntry>
+        {tree.children.length > 0 && (
```

Comment:
I fix issue where the tree still display connection when it only have root (no children).